### ISSUE TITLE
Some xml schema is not support

### ISSUE THREAD

**Issue by homerzhou** [Mermade/jgeXml#9]
example:
```
<?xml version="1.0" encoding="ISO-8859-1" ?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">

<xs:element name="test">
    <xs:complexType>
        <xs:attribute name="id" fixed="123" />
    </xs:complexType>
</xs:element>

</xs:schema>
```

'xs:attribute' is not support when there is no 'xs:sequence'->'xs:element'.
'xs:attribute' is not support 'fixed'.

**Comment by MikeRalphson** [comment]
Thanks!

**Comment by MikeRalphson** [comment]
Fixed in `v0.4.2` sincere apologies for the delay.
